filters/exclusions.txt

```diff
@@ -250,18 +250,167 @@ _campaign=$popup
+lena-miro.ru,levik.blog,livejournal.*,olegmakarenko.ru,periskop.su,shakko.ru,shiro-kino.ru,vadimrazumov.ru#?#+js(set-constant, Object.prototype.antiadblock, false)
+nova.rambler.*#?#+js(abort-on-property-read, Object.prototype.AdvManager)
+razlozhi.*#?#+js(abort-on-property-read, yaContextCb)
+razlozhi.*#?#+js(no-setTimeout-if, X-Set-Adblock)
+rp5.*#?#+js(setTimeout-defuser, xvb)
+~vp.rambler.ru,~id.rambler.ru,www.afisha.*,autorambler.*,championat.com,eda.ru,gazeta.ru,lenta.ru,letidor.*,motor.*,passion.*,quto.*,rambler.*,wmj.*#?#+js(abort-on-stack-trace, Object.prototype.getBaseFingerprint, inlineScript)
+~vp.rambler.ru,~id.rambler.ru,www.afisha.*,autorambler.*,championat.com,eda.ru,gazeta.ru,lenta.ru,letidor.*,motor.*,passion.*,quto.*,rambler.*,wmj.*#?#+js(json-prune, Blocks)
+~vp.rambler.ru,~id.rambler.ru,www.afisha.*,autorambler.*,championat.com,eda.ru,gazeta.ru,lenta.ru,letidor.*,motor.*,passion.*,quto.*,rambler.*,wmj.*#?#^script:has-text('getPrefixes')
+dzen.*,~yandex.net,yandex.*#?#+js(json-prune, direct rtb seatbid)
+frontend.vh.yandex.*#?#+js(set-constant, Object.prototype.getAutoplay, noopFunc)
+frontend.vh.yandex.*#?#+js(set-constant, Object.prototype.initialAutoplay, false)
+mail.yandex.*#?#+js(remove-attr, data-link-id, a, stay)
+mail.yandex.by,mail.yandex.kz,mail.yandex.ru,mail.yandex.uz,mail.yandex.*#?#+js(set-constant, Daria.AdvPresenter._config, {})
+music.yandex.*#?#+js(set-constant, Object.prototype.showBranding, noopFunc)
+~mail.yandex.*,~vh.yandex.ru,~yandex.net,yandex.*#?#+js(no-setTimeout-if, t.apply(n, 25)
+~mail.yandex.*,~vh.yandex.ru,~yandex.net,yandex.*#?#+js(no-setTimeout-if, Ya.Rum.getSetting, 15)
+~vh.yandex.ru,~yandex.net,yandex.*#?#+js(abort-on-property-read, Ya.loaderVer)
+~yandex.net,yandex.*#?#+js(abort-on-property-read, yaads.renderBlock)
+~yandex.net,yandex.*#?#+js(abort-on-property-read, yaads.renderWidget)
+~yandex.net,yandex.*#?#+js(cookie-remover, tvoid)
+~yandex.net,yandex.*#?#+js(remove-attr, data-counter, .serp-list a)
+~yandex.net,yandex.*#?#+js(set-constant, __AAB__, false)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype._getAdConfig, noopFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype._hasAdblock, noopFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.getHoneypot, null)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.getYaads, null)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.initWidgetBlock, falseFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.isOpenImageUnauthPopupEnabled, false)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.loaderVer, noopFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.los, null)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.useAbdBundle, false)
+~yandex.net,yandex.*#?#+js(set-constant, home.export.adb, {})
```

Comment:
Not a small list of the rules. Are they problematic? Perhaps we should consider disabling this filter?

Reply:
No. These rules has a different than filter's name purpose.
I don't know used `#?#+js()` instead of `##+js()`
They work in uBO, but no info about `#?#+js()` in their doc.

Reply:
Ok.